### DISCUSSION TITLE
Scalling multiple frames or layers

### DISCUSSION THREAD

**Discussion by Bannnnnnnnnna** [keiwando/toonsquid#385] [General]
Hi I am Trying to scale multiple frames or layers. The only way I can do this is by creating symbol but i will not be able to edit the frames except from inside symbol. Is there another way I can do this?

**Comment by mshoek** [comment]
I have a workaround, but if anybody has a better solution let me know :D

Make a new layer, and treat it as a transform layer (_in the video I named it Anim-t_). Draw something on it, otherwise, you can’t transform it. (I usually just make some “handles” in the corners.) The length should be the same, as your anim layer. In the hierarchy settings set _Anim-t_ as a parent to the anim layer. Now you can transform _Anim-t_ and it will affect your anim layer. **Make sure you are not in pixel mode in the transform settings!** If everything looks good, you can hide _Anim-t_ and merge the two layers together. The timing structure will remain the same.

**This whole thing doesn’t work with the regular dedicated transform layer, because it doesn’t have a merge option.** (Or at least I couldn't find it.)

https://github.com/keiwando/toonsquid/assets/118011378/dc2bd8b4-a7ab-4739-b174-b7856b420b58

**Reply by Bannnnnnnnnna** [reply]
Thanks a lot it worked perfectly
